src/lib/npcs.ts

```diff
@@ -696,13 +696,12 @@ export const NPC_WEARABLES: Record<NPCName, Equipped> = {
   },
   raven: {
     body: "Pale Potion",
+    onesie: "Locust King Onesie",
```

Comment:
Is this intended for Chapter launch?

Raven is a big Locust fan 😅

Reply:
Whoopsy I will remove it xD